test/lib/usd/translators/UsdImportRfMLightTest/RfMLightsTest.usda

```diff
@@ -94,12 +94,12 @@ def Xform "RfMLightsTest" (
             float inputs:intensity = 1.2
             bool inputs:normalize = 1
             float inputs:specular = 1.2
-            float shaping:cone:angle = 92
-            float shaping:cone:softness = 0.2
-            float shaping:focus = 0.2
-            color3f shaping:focusTint = (0.2, 0.2, 0.2)
-            float shaping:ies:angleScale = 1.2
-            asset shaping:ies:file = @./DiskLight_profile.ies@
+            float inputs:shaping:cone:angle = 92
+            float inputs:shaping:cone:softness = 0.2
+            float inputs:shaping:focus = 0.2
+            color3f inputs:shaping:focusTint = (0.2, 0.2, 0.2)
+            float inputs:shaping:ies:angleScale = 1.2
+            asset inputs:shaping:ies:file = @./DiskLight_profile.ies@
```

Comment:
Technically, this might actually break the test if you tried to run it in older versions of USD, but we do not actually run the RenderMan for Maya tests in this repo:
https://github.com/Autodesk/maya-usd/blob/fd878a69f4867ac88a4b8d6818e4fdd1627ae551/test/lib/usd/translators/CMakeLists.txt#L63